packages/contracts/docker/setup.sh

```
@@ -107,6 +122,33 @@ deploy_contracts() {
   popd
 }
 
+# Create a consolidated address file for easy extraction
+create_address_manifest() {
+  echo "Creating contract address manifest for easy extraction..."
+  local copied_any=false
```

Shellcheck (0.10.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)